space-cli/cmd/modules/accounts/view.go

```diff
@@ -0,0 +1,57 @@
+package accounts
+
+import (
+	"io/ioutil"
+	"os"
+	"strings"
+	"unicode/utf8"
+
+	"github.com/ghodss/yaml"
+	"github.com/olekukonko/tablewriter"
+	"github.com/sirupsen/logrus"
+
+	"github.com/spaceuptech/space-cli/cmd/model"
+)
+
+func listAccounts(accountID string, showKeys bool) error {
+	filePath := getAccountConfigPath()
+	yamlFile, err := ioutil.ReadFile(filePath)
+	if err != nil {
+		logrus.Errorf("error getting credential unable to read accounts config file - %s", err.Error())
+		return err
+	}
+
+	credential := new(model.Credential)
+	if err := yaml.Unmarshal(yamlFile, credential); err != nil {
+		logrus.Errorf("error getting credential unable to unmarshal accounts config file - %s", err.Error())
+		return err
+	}
```

Comment:
I think there already is a function for this

Reply:
The function you are referring to is a not exported function in utils creds.go but that only return a single selected account and is used in other places for that use